GVRf/Framework/src/org/gearvrf/GVRSceneObject.java

```diff
@@ -571,6 +571,36 @@ public boolean isColliding(GVRSceneObject otherObject) {
     }
 
     /**
+     * Sets the range of distances from the camera where this object will be shown.
+     *
+     * @param minRange
+     *      The closest distance to the camera rig in which this object should be shown.  Default value is 0.
+     * @param maxRange
+     *      The farthest distance to the camera rig in which this object should be shown.  Default value is Float.MAX_VALUE.
```

Comment:
Yeah, good point.  Perhaps I don't need to point that out the defaults here, but instead in getLODMinRange() and getLODMaxRange().